general/administration/reverse-proxy.md

```diff
@@ -157,7 +157,7 @@ server {
 #    }
 #    location /socket {
 #        # Proxy Jellyfin Websockets traffic
-#        proxy_pass http://SERVER_IP_ADDRESS:8096;
+#        proxy_pass http://SERVER_IP_ADDRESS:8096/;
```

Comment:
this feels wrong, it would strip `/socket` from the passed URI before proxying it to JF